local_install.R

```diff
@@ -5,6 +5,6 @@ file.arg.name <- "--file="
 script.name <- sub(file.arg.name, "", initial.options[grep(file.arg.name, initial.options)])
 pkg.dir <- paste0(dirname(script.name), "/R/pkgs/")
 
-install_local(paste0(pkg.dir, "covidcommon"), force=TRUE, upgrade="always")
-install_local(paste0(pkg.dir, "hospitalization"), force=TRUE, upgrade="always")
-install_local(paste0(pkg.dir, "report_generation"), force=TRUE, upgrade="always")
+install_local(paste0(pkg.dir, "covidcommon"), force=TRUE, upgrade="never")
```

Comment:
If we're changing this, can we please switch to install.packages?

Reply:
You can just pull the version of this script from inference, but the line is essentially: ```install.packages(<path_to_package>,type='source',repos=NULL)```